implementations/rust/ockam/ockam_api/src/control_api/backend/ticket.rs

```diff
@@ -197,30 +219,35 @@ async fn create_encoded_ticket(
 
 #[utoipa::path(
     post,
-    operation_id = "enroll_ticket",
-    summary = "Enroll a Ticket",
-    path = "/{node}/ticket",
-    tags = ["ticket"],
+    operation_id = "project_enroll",
+    summary = "Enroll to a Project using a Ticket",
+    description =
+"This API enroll a node to a Project using the provided Ticket.
+Note that this API imports the Project in the node database, but the node won't be able to use
```

Comment:
Question: how can a user restart the node?

Reply:
Theoretically with node stop/node start, but the overall DX is not yet clear-cut